src/pages/index.astro

```diff
@@ -30,26 +35,32 @@ import Button from "../components/Button.astro";
         to settle up.
       </p>
 
-      <h2 class="text-app-green mt-8 mb-4 text-3xl font-bold">How</h2>
-
-      <b>Step 1:</b> Pay Open Source maintainers.<br />
-
-      <p>
-        The minimum to participate is $2,000 per full-time employed developer
-        per year.
-      </p>
+      <h2 class="text-white mt-8 mb-4 text-3xl font-bold">How?</h2>
 
-      <b>Step 2:</b> Self-report annually:
-      <ol>
-        <li>number of full-time employed developers</li>
-        <li>payments to maintainers by platform or foundation</li>
-      </ol>
+      <div class="highlight-box">
+        <h3>Step One</h3>
+        <p>Pay Open Source maintainers.</p>
+        <p>
+          The minimum to participate is $2,000 per full-time employed developer
+          per year.
+        </p>
+      </div>
 
-      <p>Let's make it happen.</p>
+      <div class="highlight-box">
+        <h3>Step Two</h3>
+        <p>Self-report annually.</p>
+        <ol>
+          <li>number of full-time employed developers</li>
+          <li>payments to maintainers by platform or foundation</li>
+        </ol>
+      </div>
+      <div class="footer-cta-box">
+        <h2>Let's make it happen.</h2>
```

Comment:
Optional idea: if we give this `font-weight: 1.8rem`, the text aligns with the button which is pleasant.

![2024-08-05-19-36-16](https://github.com/user-attachments/assets/c7719946-883b-4d44-8220-c57d315db98a)

Reply:
ah, but it also creates a widow at default size. I don't think it's worth it.